src/main/java/bio/terra/service/snapshotbuilder/query/ExistsExpression.java

```diff
@@ -0,0 +1,25 @@
+package bio.terra.service.snapshotbuilder.query;
+
+import org.stringtemplate.v4.ST;
+
+public class ExistsExpression implements SelectExpression {
+
+  static final String GCP_TEMPLATE = "EXISTS (<subQuery>) AS <alias>";
+  static final String AZURE_TEMPLATE = "CASE WHEN EXISTS (<subQuery>) THEN 1 ELSE 0 END AS <alias>";
+
```

Comment:
can `GCP_TEMPLATE` and `AZURE_TEMPLATE` be `private?

Reply:
where do we expect these will be used outside of the class?

Reply:
I thought I might use them in the test, but I can make them private.